di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/handlers/IpvHandler.java

```diff
@@ -215,8 +215,7 @@ private URI getIpvBackchannelEndpoint(String environment) throws URISyntaxExcept
 
     public Route doCallback =
             (Request request, Response response) -> {
-                List<Map<String, Object>> mustacheData;
-                Map<String, Object> moustacheDataModel = new HashMap<>();
+                var gson = new GsonBuilder().setPrettyPrinting().create();
```

Comment:
In core we've recently removed Gson completely in favour of standardising on Jackson. Perhaps we should do the same thing here. You can make it's output pretty(er) with `new ObjectMapper().enable(INDENT_OUTPUT)`
Either way it would be good to just create the object once, rather than for every callback. Maybe set it as a static final field?